frameworks/cassandra/tests/config.py

```diff
@@ -1,17 +1,21 @@
 """ Utilties specific to Cassandra tests """
 
 import os
+import logging
+import traceback
 
 import sdk_hosts
 import sdk_jobs
 import sdk_plan
 import sdk_utils
 
+log = logging.getLogger(__name__)
 
 PACKAGE_NAME = 'cassandra'
 DEFAULT_TASK_COUNT = 3
 FOLDERED_SERVICE_NAME = sdk_utils.get_foldered_name(PACKAGE_NAME)
 DEFAULT_CASSANDRA_TIMEOUT = 600
+SERVICE_NAME = os.environ.get('SERVICE_NAME') or PACKAGE_NAME
```

Comment:
The changes to pass in SERVICE_NAME are over in soak-cluster-configurations.  FOLDERED_SERVICE_NAME is NOT used by soak-cluster-configurations at present.

I could change this to SOAK_SERVICE_NAME or something to make it clear what this escape hatch is initially for, though it seemed like a generic escape hatch to me for the service name diverging from the package name.